core/annotations/annotations.go

```diff
@@ -14,12 +14,12 @@ import (
 type Kind int
 
 const (
-	KindCharm Kind = iota
```

Comment:
Considering @jameinel gave us some guidance around this, make these hardcoded. i.e. 1,2,3,4,5 value for each kind.